src/components/gamepage/GamesPageImageGrid/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { Row, Col } from "antd";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Link } from "react-router-dom";
+// line 7-8-9 are mock datas
+import zelda from "../../../images/mockZeldaPoster.jpg";
```

Comment:
i think you need to use something refers to image for naming, for example zeldaImg etc, for alll the images

Reply:
PTAL